web-dapp/.eslintrc.yml

```diff
@@ -5,10 +5,15 @@ extends:
   - 'eslint:recommended'
   - 'plugin:react/recommended'
 parserOptions:
-  sourceType: script
+  sourceType: module
```

Comment:
We actually have a mix of `script` (server stuff) and `module` (react stuff).
I understand that `sourceType: module` is the reason we have to remove `'use strict'`, which I think is not good.
Could you try to create a separate `eslintrc` in `web-dapp/src` folder and set `sourceType: module` there, but keep `sourceType: script` here? Similar problem is described here https://stackoverflow.com/questions/41316551/eslint-ignore-specific-rule-for-a-specific-directory